src/components/SaveGameForm.js

```diff
@@ -0,0 +1,141 @@
+                <input 
+                    type="text" 
+                    name="location"
+                    id="location"
+                    placeholder="Type location ..."
+                    value={gameProfile.location}
+                    onChange={handleChange}
+                />
+                <label htmlFor="date">
+                    Date
+                </label>
+                <input 
+                    type="date" 
+                    name="date"
+                    id="date"
+                    value={gameProfile.date}
+                    onChange={handleChange}
+                />
+                <label htmlFor="players">
+                    Player(s)
+                </label>
+                <input 
+                    type="text" 
+                    name="players"
+                    id="players"
+                    placeholder="John, Jane"
+                    value={gameProfile.players}
+                    onChange={handleChange}
+                />
+                <label htmlFor="winner">
+                    Winner(s)
+                </label>
+                <input 
+                    type="text" 
+                    name="winner"
+                    id="winner"
+                    placeholder="Jane"
+                    value={gameProfile.winner}
+                    onChange={handleChange}
+                />
+                <label htmlFor="shots">
+                    Total Shots Winner(s)
+                </label>
+                <input 
+                    type="text"
+                    name="shots"
+                    id="shots"
+                    placeholder="38"
+                    value={gameProfile.shots}
+                    onChange={handleChange}
+                />
+            </InputWrapper>
+            <Button>&#10003; Save</Button>
+        </FormWrapper>
+    )
+
+
+    function handleChange(event) {
+        setGameProfile({
+            ...gameProfile,
+            [event.target.name]: event.target.value,
+            _id: uuid()
+        })
+    }
+
+    function handleSubmit(event) {
+        event.preventDefault()
+        setSavedGameProfiles([
+            gameProfile,
+            ...savedGameProfiles
+        ])
+        const form = event.target
+        setGameProfile({
+            location: '',
+            date: '',
+            players: '',
+            winner: '',
+            shots:'',
+        })
+        form.elements.location.focus()
+    }
+
+}
+
+const FormWrapper = styled.form`
+    padding: 0 10px 10px 10px;
+    display: grid;
+    place-items: center;
+    border-radius: 10px;
+    box-shadow: 0 0 10px var(--primary-medium);
+    background-color: var(--primary-medium);
+    color: var(--text-light);
+    font-size: 1rem;
+
+    Button {
+        margin-top: 15px;
+        margin-bottom: 5px;
+    }
+`
+const InputWrapper = styled.fieldset`
+    margin: 0;
+    padding: 0;
+    border: none;
+    display: grid;
```

Comment:
You don't need this here as you don't define any styles for the grid. And `display: grid;` by itself doesn't do anything :)

Reply:
I actually do need it here, in order to have all input fields and labels aligned underneath each other. Is there are more appropriate way to do so?

Reply:
I just checked in your app and I saw what you mean. I haven't really seen this kind of behavior myself, so I guess I was wrong :D I'll have a look and let you know when I find anything :)

Reply:
:D